src/Databricks/README.md

```diff
@@ -51,8 +51,9 @@ require:
   - $(this-folder)/../readme.azure.noprofile.md
 # lock the commit
 input-file:
-  - https://github.com/Azure/azure-rest-api-specs/blob/cfe9bfd432231086b92cda77a327756a90758a8f/specification/databricks/resource-manager/Microsoft.Databricks/preview/2021-04-01-preview/databricks.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/a3c9363637ad7d30407cd6dd26d280cbb166cbf9/specification/databricks/resource-manager/Microsoft.Databricks/preview/2021-04-01-preview/vnetpeering.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/53740f04d2b1f9fab8ba6b0bdc0fde9b178c5987/specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-04-01-preview/databricks.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/53740f04d2b1f9fab8ba6b0bdc0fde9b178c5987/specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-04-01-preview/vnetpeering.json
```

Comment:
These files have same commit id. Please move it to `branch`

Reply:
Done